athina/evals/function/function_evaluator.py

```diff
@@ -0,0 +1,109 @@
+
+from typing import Optional
+
+from athina.metrics.metric_type import MetricType
+import time
+from typing import Optional
+from athina.interfaces.result import EvalResult, EvalResultMetric
+from athina.helpers.logger import logger
+from athina.interfaces.athina import AthinaExperiment
+from ..base_evaluator import BaseEvaluator
+from .functions import operations
+
+class FunctionEvaluator(BaseEvaluator):
+
+    _function_name: str
+    _function_arguments: dict
+    _experiment: Optional[AthinaExperiment] = None
+    _model: str
+
+    """
+    This evaluator runs the requested Function on the given data.
+    """
+
+    @property
+    def _model(self):
+        return ""
+    
+    @property
+    def name(self):
+        return self._function_name
+
+    @property
+    def display_name(self):
+        return "Function evaluator"
+
+    @property
+    def metric_ids(self) -> str:
+        return MetricType.PASSED.value
+
+    @property
+    def default_model(self):
+        return None
+
+    @property
+    def default_function(self):
+        return "contains_any"
```

Comment:
I think you are using upper camel case everywhere else, so this should also be `ContainsAny`.

Reply:
On another note, I don't think we should have a default function.

As a user, I would be very confused about what is happening if I called `FunctionEvaluator().run(response)`.

Better to raise an error if a function name is not provided like.

Reply:
Fixed